homeassistant/components/camera/verisure.py

```diff
@@ -0,0 +1,119 @@
+"""
+Camera that loads a picture from a local file.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.verisure/
+"""
+import logging
+import os
+
+from datetime import timedelta
+from homeassistant.components.camera import Camera
+from homeassistant.components.verisure import HUB as hub
+from homeassistant.components.verisure import CONF_SMARTCAM
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Verisure File'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Camera."""
+    if not int(hub.config.get(CONF_SMARTCAM, 1)):
+        return False
+    smartcam_dict = {}
+    _LOGGER.debug('loading initial smartcam dict')
+    smartcam_dict = hub.my_pages.smartcam.get_imagelist()
+    _LOGGER.debug('smartcam_dict=%s', smartcam_dict)
+    if not smartcam_dict:
+        return False
+    device_id = list(smartcam_dict.keys())
+    file_path = os.path.join(hass.config.config_dir)
+    if not os.access(file_path, os.R_OK):
+        _LOGGER.error("file path %s is not readable", file_path)
+        return False
+    for device_id in smartcam_dict:
```

Comment:
But won't will not my code add a new camera if a new picture gets added for a new device?
What is the point of adding a camera without a picture? There is nothing to display.